packages/elements/src/components/ino-img/ino-img.tsx

```diff
@@ -122,6 +127,7 @@ export class Image implements ComponentInterface {
     if (!this.inoImgListItem) {
       this.el.querySelector('div').style.paddingTop = this.composedRatioHeight;
     }
+    this.el.querySelector('img').onerror = () => this.handleError();
```

Comment:
Could you replace the `querySelector` with an image ref?

e.g.:
```jsx
<img
  // ...
  ref={(ref) => this.imageEl = ref}
/>
```

Reply:
✅